src/components/DayPicker.jsx

```diff
@@ -80,7 +84,7 @@ export const defaultProps = {
   numberOfMonths: 2,
   orientation: HORIZONTAL_ORIENTATION,
   withPortal: false,
-  onOutsideClick() {},
+  onOutsideClick() { },
```

Comment:
please revert the addition of a space inside the body of these empty methods, throughout the file